app/src/main/res/layout/enable_exposure_notification.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Switch xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/enable_exposure_notification"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_marginTop="@dimen/padding_content"
+    android:text="Enable Exposure Notifications"
+    android:textSize="18sp"
```

Comment:
Make this a dimen?

Reply:
Actually, this is unused class that I forgot to delete. Thanks for catching it :+1: